acai-ios/SectionControllers/DetailViewSectionControllers/CustomizationOptionSectionController.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CustomizationOptionSectionController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/5/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class CustomizationOptionSectionController: ListSectionController {
+
+    var customizationOptions: [OrderCustomizationOption]!
```

Comment:
Because this section controller is supposed to render a cell for a single `OrderCustomizationOption`, you should only pass a single `OrderCustomizationOption` into the section controller instead of the entire list. Change the class variable to `var customizationOption: OrderCustomizationOption!`, and wherever you initialize this section controller make sure to set this variable.